src/Integration/TeamExplorer/ISectionController.cs

```diff
@@ -52,7 +54,7 @@ internal interface ISectionController
 
         ICommand ConnectCommand { get; }
 
-        ICommand BindCommand { get; }
+        ICommand<BindCommandArgs> BindCommand { get; }
```

Comment:
Changed the binding command property to make it explicit the type of argument that is required.
I'd like to do the same for other commands that take arguments, but I might defer that until later.